### PR TITLE
adding control and height for proper display of control panel fix issue #49

### LINKED ISSUE #49: Resizer doesn't consider the completion information
The h5p-resizing script doesn't take in consideration the completion activity information. 

- If we don't display it, we will see the control panel. 

- If we do display it, the control panel will be hidden at the bottom.

We need to add some control to adjust the height of the iframe to include the activity completion information.

### PR DESCRIPTION
**Pull request by gbarat87**

### CODE REVIEW

File: amd/src/embed.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 // When resize has been prepared tell parent window to resize.
                 H5PEmbedCommunicator.on('resizePrepared', function() {
                     H5PEmbedCommunicator.send('resize', {
-                        scrollHeight: iFrame.contentDocument.body.scrollHeight
+                        scrollHeight: iFrame.contentDocument.body.scrollHeight + ((globalThis.completion) ? 50 : 0)
```
Comment (golenkovm) [inline amd/src/embed.js]: What is 50? Where does it come from? Could it be different in different Moodle versions or Moodle themes?

Comment (gbarat87) [reply amd/src/embed.js]: That's the height of the activity completion information. Or maybe it's 60 ? i have to check again.

**Comment by gbarat87** [comment]
I have replaced the 50px by the actual automatic size of the activity completion information. And it was missing the margin-bottom size as well.

File: amd/src/embed.js [hunk 2]
```diff
@@ -22,7 +22,10 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 var resizeDelay;
                 var instance = H5P.instances[0];
                 var parentIsFriendly = false;
-
+                var completionactivity = document.getElementsByClassName('activity-header')[0];
+                var comnpletionmargin = document.defaultView.getComputedStyle(completionactivity, '')
+                    .getPropertyValue('margin-bottom');
+                var completionheight = completionactivity.scrollHeight + parseInt(comnpletionmargin, 10); // We want the first element.
```
Comment (golenkovm) [inline amd/src/embed.js:28]: Thanks @gbarat87 , this is now looking more robust. Just one more thing - what is 10? :)

File: amd/src/embed.js (hunk 1 above)
Comment (golenkovm) [reply amd/src/embed.js]: Thanks for coming up with an approach to calculate the hight dynamically. That's looking good.

File: amd/src/embed.js (hunk 2 above)
Comment (gbarat87) [reply amd/src/embed.js:28]: parseInt on base 10, but it's actually base 10 by default, so yeah, maybe I should remove that part

Comment (golenkovm) [reply amd/src/embed.js:28]: Yeah, 10 means decimal. And as you found in https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/parseint#description _this does not always default to 10_ so I'm heapy to leave it as is. Thanks

**Comment by marcusboon** [comment]
FYI, I have reverted the rebase (and put the rebased code into `stable-catalyst-fix-resize-rebase` branch) temporarily since this is currently blocking a PROD deploy. We can revisit this after the PROD deploy

**Pull request closed without merging**